plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -585,7 +585,7 @@ class GesturesPluginTest {
     verify { listener.onScaleBegin(any()) }
     presenter.removeOnScaleListener(listener)
     presenter.handleScaleBegin(scaleDetector)
-    verify(exactly = 1) { listener.onScaleBegin(any()) }
+    verify { listener.onScaleBegin(any()) }
```

Comment:
potential regression as `verify` without parameters does not take number of verifies into account, please revert back here and everywhere else

Reply:
Thanks, indeed, somehow seemed `verify` uses exactly = 1 by default. Reverted.